java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -52,7 +49,7 @@ private void scheduleFetch(long seconds) {
 
     private void fetchNewToken() {
         if (shutDown) return;
-        httpClient.post(uri, new SinglePublisher<>("", fetchExecutor), headers).subscribe(this);
+        httpClient.post(uri, new SinglePublisher<>(body, fetchExecutor), headers).subscribe(this);
```

Comment:
Ok, makes me think we never used this code...

Reply:
Yeah, we never had a body before 🤷